modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,263 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
 
+[id="pipelines-new-features-1-8_{context}"]
+=== Pipelines
+
 :FeatureName: Running {pipelines-title} on ARM hardware
-[id="running-pipelines-on-arm-hardware-1-8_{context}"]
-=== {FeatureName}
+* {FeatureName} With this update, you can run {pipelines-title} GA 1.8 and later on an {product-title} cluster that is running on ARM hardware. This includes support for `ClusterTask` resources and the `tkn` CLI tool.
++
+include::snippets/technology-preview.adoc[]
+// Shubham Minglani
+
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible. This capability includes support for repositories on GitHub, GitLab, and BitBucket.
+// (link:https://issues.redhat.com/browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+* Previously, as a developer, you could have only one pipeline run in your `.tekton` directory for each Git event, such as a pull request. With this update, you can have all the pipeline runs in your `.tekton` directory match specified events. The web console displays the status and reports of the runs. If you are using WebVCS, the reports include comments.
+// (link:https://issues.redhat.com/browse/SRVKP-1781[SRVKP-1781 Multiple Pipelines runs match on events])
+// Chmouel Boudjnah
+
+* With this update, as a developer, you get a notification from the `tkn-pac` CLI tool if you try to add a duplicate repository to a {pac} run. When you enter `tkn pac repo create`, each repository must have a unique URL. This notification also helps prevent hijacking exploits.
+// (link:https://issues.redhat.com/browse/SRVKP-1758[SRVKP-1758 Pipeline as code: validate repository URL uniqueness])
+// Shivam Mukhade
+
+* This update implements `Step` and `Sidecar` overrides for `TaskRun` resources.
+// link:https://github.com/tektoncd/pipeline/pull/4598
+
+* This update adds minimal `TaskRun` and `Run` statuses within `PipelineRun` statuses.
+// link:https://github.com/tektoncd/pipeline/pull/4739
+
+* With this update, the graceful termination of pipeline runs feature is promoted from a beta feature to a stable feature. As a result, the previously deprecated `PipelineRunCancelled` status remains deprecated and is planned to be removed in a future release.
+// link:https://github.com/tektoncd/pipeline/pull/4668
+
+* With this update, you can specify the workspace for a pipeline task by using the name of the workspace. This change makes it easier to specify a shared workspace for a pair of `Pipeline` and `PipelineTask` resources. You can also continue to map workspaces explicitly.
+// link:https://github.com/tektoncd/pipeline/pull/4887
+
+* With this update, parameters in embedded specifications are propagated without mutations.
+//https://github.com/tektoncd/pipeline/pull/4845
```

Comment:
Should this have the `link:` ?

Reply:
Updated. Thanks. (Currently, we don't publish these links for customers to see.)